hogvm/python/execute.py

```diff
@@ -248,7 +248,7 @@ def capture_upvalue(index) -> dict:
             case Operation.GET_GLOBAL:
                 chain = [pop_stack() for _ in range(next_token())]
                 if globals and chain[0] in globals:
-                    push_stack(deepcopy(get_nested_value(globals, chain)))
+                    push_stack(deepcopy(get_nested_value(globals, chain, True)))
```

Comment:
Ooh i didn't realise this was only the case for globals 🤔 In which case I think it makes 100% sense this way.

Reply:
That said - does it make the `?.` accessor redundant?

Reply:
Yeah, it kind of does. I thought non-global property access threw if you accessed something on a null, but I guess I must've changed it somewhere. 

I'll see if I need to remove/deprecate this fully, or if there's some place where it is still useful